File: backend/vs_core/UPC.Bagueteria.Domain/UPC.Bagueteria.Domain/Models/EntityResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UPC.Bagueteria.Domain.Models
{
    public class EntityResponse
    {
        public EntityResponse()
        {
            _isSuccess = false;
            _errorCode = "0000";
            _errorMessage = "";
            _data = null;
        }
        private Boolean _isSuccess;
        private String _errorCode;
        private String _errorMessage;
        private Object _data;

        public bool IsSuccess { get => _isSuccess; set => _isSuccess = value; }
        public string ErrorCode { get => _errorCode; set => _errorCode = value; }
        public string ErrorMessage { get => _errorMessage; set => _errorMessage = value; }
        public object Data { get => _data; set => _data = value; }
    }
}

File: backend/java_spring/Biscuits/src/main/java/com/upc/bakery/biscuits/models/EntityResponse.java
package com.upc.bakery.biscuits.models;

public class EntityResponse {
    private boolean IsSuccess;
    private String ErrorCode;
    private String ErrorMessage;
    private Object Data;

    public EntityResponse() {
        this.IsSuccess = false;
        this.ErrorCode="";
        this.ErrorMessage="";
        this.Data=null;
    }

    public boolean isSuccess() {
        return IsSuccess;
    }

    public void setSuccess(boolean success) {
        IsSuccess = success;
    }

    public String getErrorCode() {
        return ErrorCode;
    }

    public void setErrorCode(String errorCode) {
        ErrorCode = errorCode;
    }

    public String getErrorMessage() {
        return ErrorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        ErrorMessage = errorMessage;
    }

    public Object getData() {
        return Data;
    }

    public void setData(Object data) {
        Data = data;
    }
}
